src/WebJobs.Script/NullHostedService.cs

```diff
@@ -6,11 +6,11 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Hosting;
 
-namespace Microsoft.Azure.WebJobs.Script.WebHost
+namespace Microsoft.Azure.WebJobs.Script
 {
     internal sealed class NullHostedService : IHostedService
     {
-        private static readonly Lazy<NullHostedService> _instance = new Lazy<NullHostedService>(new NullHostedService());
+        private static readonly Lazy<NullHostedService> _instance = new Lazy<NullHostedService>(() => new NullHostedService());
```

Comment:
Not sure we're saving much with this here. Instance is pretty much guaranteed to be used on this type to trigger static initialization.

Reply:
The `Lazy<T>(T)` constructor doesn't exist in netstandard2.0 (but does exist in netcore3.1). So when I moved this class from Script.WebHost -> Script, I had to change it as WebHost targets 3.1 but Script targets netstandard2.0.